pkg/cmd/cobra/cobra.go

```diff
@@ -232,6 +232,7 @@ func GetTraceeRunner(c *cobra.Command, version string) (cmd.Runner, error) {
 	}
 
 	cfg.Policies = policies
+	policy.Snapshots().Store(cfg.Policies)
```

Comment:
This is a very good singleton practice/way (instead of saving instanced objects in Tracee singleton). Im just afraid that there should be an effort to make all others like this one somehow. There are many instanciated objects that could use the same approach.

Reply:
Nice. We can expand this pattern in the future.